notification.py

```diff
@@ -622,8 +626,54 @@ def notify(self, pokemon, time_of_day):
                     n=name, s=mean))
                 return False
 
-        notified = Notification(pokemon_id, coordinates, time_till_hidden, iv, moves, iv_score, time_of_day).notify()
-        if notified:
+        whpushed = False
+        if config.WEBHOOKS:
+            whpushed = self.webhook(pokemon, time_till_hidden)
+
+        notified = False
+        if config.PB_API_KEY or config.TWITTER_ACCESS_KEY:
+            notified = Notification(pokemon_id, coordinates, time_till_hidden, iv, moves, iv_score, time_of_day).notify()
+
+        if notified or whpushed:
             self.last_notification = monotonic()
             self.recent_notifications.append(encounter_id)
-        return notified
+        return notified or whpushed
+
+    def webhook(self, pokemon, time_till_hidden):
+        """ Send a notification via webhook
+        """
+        if isinstance(time_till_hidden, (tuple, list)):
+            time_till_hidden, _ = time_till_hidden
+
+        data = {
+            'type': "pokemon",
+            'message': {
+                "encounter_id": pokemon['encounter_id'],
+                "pokemon_id": pokemon['pokemon_id'],
+                "last_modified_time": pokemon['seen'] * 1000,
+                "spawnpoint_id": pokemon['spawn_id'],
+                "latitude": pokemon['lat'],
+                "longitude": pokemon['lon'],
+                "disappear_time": pokemon['seen'] + time_till_hidden,
+                "time_until_hidden_ms": time_till_hidden * 1000
+            }
+        }
+
+        if pokemon.get('individual_attack'):
```

Comment:
Either do
```python
pokemon.get('individual_attack') is not None:
```
or remove the `if` statement and use the form `pokemon['individual_attack']` with this block wrapped in `try:` `except KeyError`.

Because `0`s fail the truth test.